.github/workflows/start-self-hosted-runner/action.yml

```diff
@@ -4,25 +4,28 @@ inputs:
     aws-region:
         description: AWS Region, e.g. us-east-1
         required: true
-    aws-access-key-id:
+    role-session-name:
```

Comment:
Please update descriptions to match

Reply:
its a draft pr, I need the workflows so its not on draft